medallion/test/test_backends.py

```diff
@@ -914,56 +819,39 @@ def test_get_version_limit(backend):
     assert r.headers['X-TAXII-Date-Added-Last'] == '2017-12-31T13:49:53.935000Z'
 
 
-def test_get_version_spec_version(backend):
+def get_version_spec_version(backend, getstr):
     r = backend.client.get(
-        test.GET_OBJECTS_EP + "malware--c0931cc6-c75e-47e5-9036-78fabc95d4ec/versions?match[spec_version]=2.0",
+        test.GET_OBJECTS_EP + getstr,
```

Comment:
Since all the tests that use this function contain `malware--c0931cc6-c75e-47e5-9036-78fabc95d4ec/versions` in their url, maybe include it here so it doesn't have to be passed in as part of `getstr`.

Reply:
don't see this in my current code, i think i already fixed it